app/upgrades/v17/constants.go

```diff
@@ -17,3 +18,204 @@ var Upgrade = upgrades.Upgrade{
 		Deleted: []string{},
 	},
 }
+
```

Comment:
This whole file should be verified against mainnet by the reviewer

Reply:
The only thing that needs to be verified is the pool numbers now, since the other values are auto populated at time of execution

Reply:
checked the pools and it looks good, (all non-eth tokens) 

ps: i'm surprised we're not migrating osmo/atom pool